src/ex4/server/clients/pokemon_client.js

```diff
@@ -1,29 +1,24 @@
 // The Pokemon Client (using axios) goes here
 import axios from "axios"
-import { pokemonExistsInTodoList } from "../helper/pokemonHelper.js"
 
 export async function getPokemon(num) {
     const url = `https://pokeapi.co/api/v2/pokemon/${num}`
-    const pokeName = await axios.get(url).then(res => {
-        return res
-    }).catch(e => {
-        return 0
-    })
-    return pokeName
+    const pokeName = await axios.get(url)
+    console.log(pokeName)
+    if (pokeName.status === 200) {
+        return pokeName
+    }
+    return undefined
```

Comment:
returning undefined doesn't have any value. if you need to return something I would return null.